server/src/test/test.js

```diff
@@ -0,0 +1,21 @@
+import chai, { expect } from "chai";
+import chaiHttp from "chai-http";
+import app from "/../NodeProjects/My meal App/My-Meal-App/app";
+
+chai.use(chaiHttp);
+chai.should();
+
+describe("menu", () => {
+  describe("GET Menu", () => {
+    it("should get the menu for the day", done => {
+      chai
+        .request(app)
+        .get("/api/v1/menu")
+        .end((err, res) => {
+          res.should.have.status(200);
+          res.body.should.be.a("object");
```

Comment:
This line doesn't test for the appropriate response. You want to assert that you get back a list of menu with 0 or more items.

Reply:
boss pls can u throw more light on this please